cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -477,6 +477,10 @@ public boolean hasOngoingExecution() {
     return _hasOngoingExecution;
   }
 
+  public boolean hasOngoingPartitionReassignments() {
```

Comment:
Can we add a JavaDoc for the public function?
In particular, can we emphasize that this function is prone to provide false negatives? -- i.e. even when Cruise Control is in the process of executing replica moves, there could be quiet periods with no replica being reassigned (as described on the previous feedback above)?